frontend/sass/norent/styles.scss

```diff
@@ -61,4 +76,143 @@ $jf-navbar-content: $black;
       color: $link;
     }
   }
+
+  // Default spacing between paragraphs
+  p:not(:last-child) {
+    margin-bottom: 0.5rem;
+  }
+
+  // Set all Bulma container classes to have max width:
+  .container {
+    max-width: 800px;
+    &.jf-tight-container {
+      max-width: 600px;
+    }
+    &.jf-wide-container {
+      max-width: 1000px;
+    }
+  }
+
+  // Make all media icons have vertical centering
+  // as well as smaller margins on mobile
+  .media .media-left,
+  .media .media-right {
+    margin-top: auto;
+    margin-bottom: auto;
+    @media screen and (max-width: $tablet) {
+      figure {
+        margin-left: 1rem;
+        margin-right: 1rem;
+      }
+    }
+  }
+}
+
+// CUSTOM CLASSES
+
+@mixin is-horizontal-center {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.jf-space-below {
+  margin-bottom: 2rem;
+}
```

Comment:
If you do end up putting this in a separate helpers SCSS file, I recommend actually putting the amount of margin into the title of the class so it's `jf-space-below-2em`.  That way someone reading the HTML/JSX will know exactly what it means... The only downside of course is that if we change it to be 3em, we'll have to change the class name too, but on the plus side it will also dissuade anyone from arbitrarily changing it just so they can fix the spacing on one particular instance (which could break spacing on other instances).

Reply:
Good call! I'm going to do that right now so I don't forget to change it later.